frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx

```diff
@@ -769,32 +769,24 @@ export function useClusterProviderColumn(): IAcmTableColumn<Cluster> {
 
 export const getControlPlaneString = (cluster: Cluster, t: TFunction<string, undefined>) => {
   const clusterHasControlPlane = () => {
-    const nodeList = cluster.nodes?.nodeList
-    const roleList = nodeList?.map((node: NodeInfo) => getRoles(node))
-    const hasControlPlane = roleList?.filter((str) => {
-      return str.indexOf('control-plane') > -1
-    })
-    return hasControlPlane ? hasControlPlane.length > 0 : false
+    return cluster.nodes?.nodeList?.some((node: NodeInfo) => getRoles(node).includes('control-plane')) || false
   }
 
+  const isHosted =
+    cluster.isHostedCluster ||
+    cluster.isHypershift ||
+    (cluster.distribution?.displayVersion?.includes('ROSA') && !clusterHasControlPlane())
+
```

Comment:
Let's also define `const isHub` to be `cluster.name === 'local-cluster' || cluster.isRegionalHubCluster`
And then the return statements can be further simplified:

1. isHub && isHosted -> Hub, Hosted
2. isHub -> Hub
3. isHosted -> Hosted
4. else Standalone